api/sip/client.py

```diff
@@ -315,14 +316,16 @@ def connect(self):
                 self.connection = self.make_secure_connection()
             else:
                 self.connection = self.make_insecure_connection()
+            
+            self.connection.settimeout(12)
```

Comment:
I moved this from outside the try statement to before the `connect()` call otherwise it would just stall and the tests would never run. Also, is this suppose to represent 12 seconds? The docs say so but it always seems to timeout at twice the number input.

Reply:
That's weird. Maybe what you're seeing is that there are two requests and they both have to time out for an error to be raised out of the SIP2 client?

Setting the timeout immediately makes sense.